reflex/vars/base.py

```diff
@@ -583,12 +563,6 @@ def create(
         if not isinstance(value, str):
             return LiteralVar.create(value, _var_data=_var_data)
```

Comment:
we should remove this too

Reply:
the same exact line is just below. with the intervening fallback stuff taken out, there's no reason to create the LiteralVar early for `str` types